uuid-js/index.d.ts

```diff
@@ -0,0 +1,43 @@
+// Type definitions for uuid-js 0.7
+// Project: http://github.com/pnegri/uuid-js
+// Definitions by: Mohamed Hegazy <https://github.com/mhegazy>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+export = uuid_js;
+declare class uuid_js {
+    constructor();
```

Comment:
This isn't documented. You're probably meant to use the static constructors instead.